modules/rdma-creating-persistent-naming-rules.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-creating-persistent-naming-rules_{context}"]
+
+= Creating persistent naming rules
+
+In some cases, device names won't persist following a reboot. For example, on R760xa systems 
+Mellanox devices might be renamed after a reboot. You can avoid this problem by using a `MachineConfig` to set persistence.
```

Comment:
Nit - per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block) we should not hard wrap our lines:
```suggestion
In some cases, device names won't persist following a reboot. For example, on R760xa systems Mellanox devices might be renamed after a reboot. You can avoid this problem by using a `MachineConfig` to set persistence.
```

Reply:
Also, adding a noun after `MachineConfig` might provide more clarity to users.

Reply:
Changed.